samples/sample_common/include/brc_routines.h

```diff
@@ -243,12 +240,12 @@ class AVGBitrate
         {
             if (bPanic || bSH)
             {
-                m_maxWinBitsLim = IPP_MAX(IPP_MIN( (GetLastFrameBits(windowSize, false) + m_maxWinBits)/2, m_maxWinBits), GetMaxWinBitsLim());
+                m_maxWinBitsLim = std::max(std::min( (GetLastFrameBits(windowSize, false) + m_maxWinBits)/2, m_maxWinBits), GetMaxWinBitsLim());
             }
             else
             {
                 if (recode)
-                    m_maxWinBitsLim = IPP_MIN(IPP_MAX(GetLastFrameBits(windowSize,false) + GetStep()/2, m_maxWinBitsLim), m_maxWinBits);
+                    m_maxWinBitsLim = std::max(std::min(GetLastFrameBits(windowSize, false) + GetStep()/2, m_maxWinBits), m_maxWinBitsLim);
```

Comment:
is min/max reorder by intent?

Reply:
yes, now layout of line 248 matches to layout of line 243